### PR TITLE
Add application CRUD audit events.

### PR DESCRIPTION
**Pull request by mdwn**
Application audit events for CRUD events are now recognized in the frontend as opposed to being displayed as 'unknown.'

This will address #9490.

### CODE REVIEW

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 1]
```diff
@@ -752,6 +752,24 @@ export const formatters: Formatters = {
     desc: 'Database Deleted',
     format: ({ user, name }) => `User [${user}] deleted database [${name}]`,
   },
+  [eventCodes.APPLICATION_CREATED]: {
+    type: 'app.create',
```
Comment (zmb3) [inline packages/teleport/src/services/audit/makeEvent.ts:756]: Here we have `app.create` but in the fixture above we have `app.created`. What's the difference?

Comment (mdwn) [reply packages/teleport/src/services/audit/makeEvent.ts:756]: Whoops. That would be a typo from me, good catch.

File: packages/teleport/src/services/audit/types.ts [hunk 2]
```diff
@@ -215,6 +218,24 @@ export type RawEvents = {
...
+    }
+  >;
+  [eventCodes.APPLICATION_UPDATED]: RawEvent<
+    typeof eventCodes.APPLICATION_UPDATED,
+    {
+      name: string;
+    }
+  >;
+  [eventCodes.APPLICATION_DELETED]: RawEvent<
+    typeof eventCodes.DATABASE_DELETED,
```
Comment (r0mant) [inline packages/teleport/src/services/audit/types.ts:234]: Should this be `APPLICATION_DELETED`?

File: packages/teleport/src/services/audit/makeEvent.ts (hunk 1 above)
Comment (mdwn) [reply packages/teleport/src/services/audit/makeEvent.ts:756]: This is fixed, but I'm sort of confused why it worked -- is the `type` not used for anything outside of the details page?

File: packages/teleport/src/services/audit/types.ts [hunk 3]
```diff
@@ -49,6 +49,9 @@ export const eventCodes = {
   APP_SESSION_CHUNK: 'T2008I',
   APP_SESSION_START: 'T2007I',
   APP_SESSION_END: 'T2011I',
+  APPLICATION_CREATED: 'TAP03I',
+  APPLICATION_UPDATED: 'TAP04I',
+  APPLICATION_DELETED: 'TAP05I',
```
Comment (r0mant) [inline packages/teleport/src/services/audit/types.ts:54]: This is definitely a nitpick, but can we name these `APP_CREATED`, `APP_UPDATED`, etc. so they're consistent with event codes that are already defined here?

File: packages/teleport/src/services/audit/types.ts (hunk 2 above)
Comment (mdwn) [reply packages/teleport/src/services/audit/types.ts:234]: Yep, that's correct. Bad copy-paste.

File: packages/teleport/src/services/audit/types.ts (hunk 3 above)
Comment (mdwn) [reply packages/teleport/src/services/audit/types.ts:54]: Sure, no prob!

**Review by r0mant — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Comment by zmb3** [comment]
@mdwn Please reach out if you need help with the backport process. We want to make sure this fix makes it into all supported versions.

**Comment by mdwn** [comment]
> @mdwn Please reach out if you need help with the backport process. We want to make sure this fix makes it into all supported versions. 

Ack, definitely will do -- I'll reach out in the morning!

**Pull request merged**